### PR TITLE
Prevent freezing webhook

### PR DESCRIPTION
**Pull request by nishimaki10**
@RocketChat/core 

Even if a crazy scripts in webhooks.
e.g. `while(true) { ... }`

### CODE REVIEW

File: packages/rocketchat-integrations/server/api/api.coffee [hunk 1]
```diff
@@ -167,7 +167,18 @@ executeIntegrationRest = ->
...
+				s: s
+				console: console
+				Store:
+					set: (key, val) ->
+						return store[key] = val
+					get: (key) ->
+						return store[key]
+				script: script
+				request: request
+			result = vm.runInNewContext('script.process_incoming_webhook({ request: request })', sandbox, { timeout: 3000 })
```
Comment (rodrigok) [inline packages/rocketchat-integrations/server/api/api.coffee]: Why did you change the method name from `process_incoming_request` to `process_incoming_webhook`?

Comment (nishimaki10) [inline packages/rocketchat-integrations/server/api/api.coffee]: Oh... It is my mistake.
